mergin/test/test_client.py

```diff
@@ -123,7 +124,7 @@ def test_create_remote_project_from_local(mc):
     assert project_info['name'] == test_project
     assert project_info['namespace'] == API_USER
 
-    versions = mc.project_versions(project)
+    versions = mc.project_versions(project)["versions"]
```

Comment:
hmm this looks like we are breaking API of the mergin client if it is needed to access `"version"` in the returned dict...

Reply:
we do, I really want to disable using non-paginating endpoints, I did grep for this function in py-client, it was not used, I will double check plugin